google/cloud/pubsub/samples/client_samples.cc

```diff
@@ -0,0 +1,351 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "google/cloud/pubsub/blocking_publisher.h"
+#include "google/cloud/pubsub/samples/pubsub_samples_common.h"
+#include "google/cloud/pubsub/schema_admin_client.h"
+#include "google/cloud/pubsub/subscriber.h"
+#include "google/cloud/pubsub/subscription_admin_client.h"
+#include "google/cloud/pubsub/subscription_builder.h"
+#include "google/cloud/pubsub/topic_admin_client.h"
+#include "google/cloud/credentials.h"
+#include "google/cloud/internal/getenv.h"
+#include "google/cloud/internal/random.h"
+#include <fstream>
+#include <string>
+#include <vector>
+
+namespace {
+
+using ::google::cloud::internal::GetEnv;
+using ::google::cloud::pubsub::examples::RandomSubscriptionId;
+using ::google::cloud::pubsub::examples::RandomTopicId;
+
+void PublisherSetEndpoint(std::vector<std::string> const& argv) {
+  namespace examples = ::google::cloud::testing_util;
```

Comment:
The other samples (rightfully, IMO) include `google/cloud/testing_util/example_driver.h` directly.

Reply:
Done.